### PR TITLE
New package: termrec-0.18

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

File: srcpkgs/termrec/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Template file for 'termrec'
+pkgname=termrec
+version=0.18
+revision=1
+build_style=gnu-configure
+makedepends="libcurl-devel"
+depends="libtermrec"
```
Comment (Chocimier) [inline srcpkgs/termrec/template]: Automatically found:

=> termrec-0.18_1: running pre-pkg hook: 04-generate-runtime-deps ...

   SONAME: libtty.so.1 <-> libtermrec-0.18_1

Comment (abenson) [reply srcpkgs/termrec/template]: I had it in mind there was some weird behavior with depending on sub-packages. Fixed.

Comment (xtraeme) [reply srcpkgs/termrec/template]: the depends line is unnecessary

Comment (abenson) [reply srcpkgs/termrec/template]: I could've sworn I removed that

**Pull request merged**